draft-ietf-jsonpath-base.md

```diff
@@ -391,6 +391,10 @@ of bracket notation. Examples and descriptions use shorthands where convenient.
 
 ### Selectors
 
+A name selector, e.g. `name`, selects a named child of an object.
```

Comment:
Does this need quotes (e.g. `'name'`)?  Might be confusing with the `3` that doesn't have them.

Reply:
Nice catch! Fixed.